blocks/multicolor-accordion/multicolor-accordion.js

```diff
@@ -0,0 +1,53 @@
+export default async function decorate(block) {
+  const accordionItems = block.querySelectorAll(':scope > div > div');
+  accordionItems.forEach((accordionItem) => {
+    const nodes = accordionItem.children;
+
+    const titleText = nodes[0];
+    const rest = Array.prototype.slice.call(nodes, 1);
+
+    const titleDiv = document.createElement('div');
+    const detailsBtn = document.createElement('button');
+    detailsBtn.innerText = 'Details';
```

Comment:
we need this text at least configurable via placeholder if not overwrite-able in the block